Testing/functional/tests/cypress/integration/ui/timeline/bcCancerScreening.js

```diff
@@ -0,0 +1,41 @@
+const { AuthMethod } = require("../../../support/constants");
+
+describe("BC Cancer Screening cards", () => {
+    function testCard(cardTitle, cardButtonText) {
+        cy.get("[data-testid=timelineCard")
+            .filter(`:contains(${cardTitle})`)
+            .first()
+            .within(() => {
+                cy.get("[data-testid=bccancerscreeningTitle]").click({
+                    force: true,
+                });
+                cy.get("[data-testid=bc-cancer-screening-download-button]")
+                    .contains(cardButtonText, { matchCase: false })
+                    .should("exist");
+            });
+    }
+
+    before(() => {
+        cy.intercept("GET", "**/PatientData/*?patientDataTypes=*", {
+            fixture: "PatientData/bcCancerTypes.json",
+        });
+        cy.configureSettings({
+            datasets: [
+                {
+                    name: "bcCancerScreening",
+                    enabled: true,
+                },
+            ],
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+    });
+    it("Should display different cards for different types", () => {
```

Comment:
I see there is a UI test to show the two event types but is there an e2e test to validate the download of the files for the two event types?

Reply:
This may be brittle though but I'll add it.